cdap-watchdog/src/main/java/co/cask/cdap/logging/write/FileMetaDataManager.java

```diff
@@ -190,50 +161,6 @@ public Void call() throws Exception {
     });
   }
 
-  /**
-   * // TODO refactor this with the above method after Log handler changes.
-   * Returns a list of log files for a logging context.
-   *
-   * @param logPathIdentifier logging context identifier.
-   * @return List of {@link LogLocation}
-   */
-  public List<LogLocation> listFiles(final LogPathIdentifier logPathIdentifier) throws Exception {
```

Comment:
So we are only moving code used by new `CDAPLogAppender` remainder still remains in `FileMetadataManager`? Because we are planning to eventually remove that class?